python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -231,33 +231,44 @@ def search_summary(self):
         return self.hposearcher.search_summary()
 
     def quantize(self, pl_model,  # remove the type requirement for type checking
+                 precision='int8',
+                 accelerator=None,
+                 method='fx',
                  calib_dataloader: DataLoader = None,
                  val_dataloader: DataLoader = None,
                  metric: Optional[Metric] = None,
                  backend='inc',
                  conf: Optional[str] = None,
-                 framework='pytorch_fx',
                  approach='static',
                  tuning_strategy='bayesian',
                  accuracy_criterion: dict = {'relative': 0.99, 'higher_is_better': True},
                  timeout=0,
                  max_trials=1,
-                 return_pl=True
+                 input_sample=None,
+                 return_pl=False
                  ):
         """
         Calibrate a Pytorch-Lightning model for post-training quantization.
 
-        :param pl_model:       A Pytorch-Lightning model to be quantized.
+        :param pl_model:        A Pytorch-Lightning model to be quantized.
```

Comment:
Does it have to be a Pytorch-Lightning model? Can we support standard PyTorch model?

Reply:
Yes, a pytorch model is acceptable. Will change the doc.